### PR TITLE
Add experimental feature 'v0.5.0'

### PR DESCRIPTION
**Pull request by axic**

### CODE REVIEW

File: libsolidity/ast/ExperimentalFeatures.h [hunk 1]
```diff
@@ -45,6 +46,7 @@ static const std::map<std::string, ExperimentalFeature> ExperimentalFeatureNames
 {
 	{ "SMTChecker", ExperimentalFeature::SMTChecker },
 	{ "ABIEncoderV2", ExperimentalFeature::ABIEncoderV2 },
+	{ "v0.5.0", ExperimentalFeature::V050 },
```
Comment (axic) [inline libsolidity/ast/ExperimentalFeatures.h:49]: Not sure which one we're going for, `v0.5.0`, `v050`, `V0.5.0`, .. ?

Comment (chriseth) [reply libsolidity/ast/ExperimentalFeatures.h:49]: I think something like `v0.5.0` is fine, but do we want to have the trailing `0`? Perhaps we should also define these features somewhere, i.e. what exactly does it activate?

Comment (axic) [reply libsolidity/ast/ExperimentalFeatures.h:49]: I think we can have a documentation section about them if needed, but wouldn't put `v0.5.0` in there, it is a bit more like having an `#ifdef` so we can merge breaking changes in preparation for the breaking release.

**Comment by axic** [comment]
It is only `testrpc` failing.

**Pull request merged**